src/libraries/System.Private.Xml/src/System/Xml/Extensions/ExtensionMethods.cs

```diff
@@ -1,18 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Reflection;
-
-
 namespace System.Xml.Extensions
 {
     internal static class ExtensionMethods
     {
+        private static readonly char[] s_trimChars = new char[] { ' ', '\t', '\n', '\r' };
```

Comment:
```suggestion
        private static readonly char[] s_trimChars = new char[] { ' ', '\t', '\n', '\r' };

```

Reply:
@krwq I've checked once more and this function looks same as XmlConvert.ToUri. Maybe it is better to delete this and use other one instead of optimizing it ?

Reply:
if you think they're exactly the same that sounds like a good idea.

Reply:
I will remove it as I don't see any difference:
` private static readonly char[] s_trimChars = new char[] { ' ', '\t', '\n', '\r' };

        internal static Uri ToUri(string s)
        {
            if (s != null && s.Length > 0)
            { //string.Empty is a valid uri but not "   "
                s = s.Trim(s_trimChars);
                if (s.Length == 0 || s.IndexOf("##", StringComparison.Ordinal) != -1)
                {
                    throw new FormatException(SR.Format(SR.XmlConvert_BadFormat, s, "Uri"));
                }
            }
            if (!Uri.TryCreate(s, UriKind.RelativeOrAbsolute, out Uri? uri))
            {
                throw new FormatException(SR.Format(SR.XmlConvert_BadFormat, s, "Uri"));
            }

            return uri;
        }`

and

internal static Exception? TryToUri(string s, out Uri? result)
        {
            result = null;

            if (s != null && s.Length > 0)
            { //string.Empty is a valid uri but not "   "
                s = TrimString(s);
                if (s.Length == 0 || s.IndexOf("##", StringComparison.Ordinal) != -1)
                {
                    return new FormatException(SR.Format(SR.XmlConvert_BadFormat, s, "Uri"));
                }
            }
            if (!Uri.TryCreate(s, UriKind.RelativeOrAbsolute, out result))
            {
                return new FormatException(SR.Format(SR.XmlConvert_BadFormat, s, "Uri"));
            }

            return null;
        }